lib/JITIDL/JITTypes.h

```diff
@@ -77,7 +77,7 @@ typedef unsigned char boolean;
 #define __JITTypes_h__
 
 // TODO: OOP JIT, how do we make this better?
-const int VTABLE_COUNT = 48;
+const int VTABLE_COUNT = 49;
```

Comment:
Huh- didn't know this existed...is there a compile assert or test that exists to remind us to update this?

Reply:
Yeah there is a compile assert